File: packages/app-launcher/ios/Plugin/Classes/Results/CanOpenUrlResult.swift
import Foundation
import Capacitor

@objc public class CanOpenUrlResult: NSObject, Result {
    let value: Bool

    init(value: Bool) {
        self.value = value
    }

    @objc public func toJSObject() -> AnyObject {
        var result = JSObject()
        result["value"] = value
        return result as AnyObject
    }
}

File: packages/app-launcher/android/src/main/java/io/capawesome/capacitorjs/plugins/applauncher/classes/results/CanOpenUrlResult.java
package io.capawesome.capacitorjs.plugins.applauncher.classes.results;

import androidx.annotation.NonNull;
import com.getcapacitor.JSObject;
import io.capawesome.capacitorjs.plugins.applauncher.interfaces.Result;

public class CanOpenUrlResult implements Result {

    private final boolean value;

    public CanOpenUrlResult(boolean value) {
        this.value = value;
    }

    @Override
    @NonNull
    public JSObject toJSObject() {
        JSObject result = new JSObject();
        result.put("value", value);
        return result;
    }
}
